src/roles/kserve-tests/e2e-test/config.yaml

```
@@ -0,0 +1,69 @@
+role:
+  created_date: "20250620"
+  name: kserve-odh-e2e-test
+  description: |
+    This role help to run kserve e2e test. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)